account/accounts.go

```diff
@@ -61,6 +61,11 @@ type RecoverParams struct {
 	Message   string `json:"message"`
 	Signature string `json:"signature"`
 }
+type AccountInterface interface {
+	GetVerifiedWalletAccount(db *accounts.Database, address, password string) (*SelectedExtKey, error)
+	Sign(rpcParams SignParams, verifiedAccount *SelectedExtKey) (result types.HexBytes, err error)
+	Recover(rpcParams RecoverParams) (addr types.Address, err error)
+}
```

Comment:
Nice use of interface. Would you be able to add the go docs for this?

```suggestion

// AccountInterface represents something and is used for things
type AccountInterface interface {
	GetVerifiedWalletAccount(db *accounts.Database, address, password string) (*SelectedExtKey, error)
	Sign(rpcParams SignParams, verifiedAccount *SelectedExtKey) (result types.HexBytes, err error)
	Recover(rpcParams RecoverParams) (addr types.Address, err error)
}
```

Reply:
Sure! Btw this PR on hold until next week